docs.bruno/Create req. (POST).bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Create req. (POST)
+  type: http
+  seq: 2
+}
+
+post {
+  url: 
```

Comment:
The URL for the POST request is missing. Please provide the correct URL for the POST request in the Bruno file. [important]